empathy.lic

```diff
@@ -0,0 +1,51 @@
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#empathy
+=end
+
+custom_require.call(%w(common common-travel drinfomon))
+
+class Empathy
+  include DRC
+  include DRCT
+
+  def initialize
+    settings = get_settings
+    @stationary_skills_only = settings.crossing_training_stationary_skills_only
+    @hometown = settings.hometown
+
+    train_empathy
+  end
+  
+  def check_research
+    return unless @researching

```

Comment:
Would removing @researching all together work for it, as perc and perc health both will not break research?